services/order.py

```diff
@@ -0,0 +1,35 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+from db.models import Ticket, Order
+
+
+def create_order(
+    tickets: list[dict], username: str, date: str = None
+) -> None:
+    with transaction.atomic():
+        user = get_user_model().objects.get(username=username)
+        order = Order.objects.create(user=user)
+
+        if date:
+            order.created_at = date
+            order.save()
+
+        [
+            Ticket.objects.create(
+                movie_session_id=ticket["movie_session"],
+                row=ticket["row"],
+                seat=ticket["seat"],
+                order=order
+            )
+            for ticket in tickets
+        ]
+
+
+def get_orders(username: str = None) -> QuerySet:
+    orders = Order.objects.order_by("-user")
```

Comment:
Ordering is not needed here. The tests should pass without it on GitHub even if they fail locally; there was a bug in tests that is now fixed.

Reply:
And this one.

Reply:
Yes. I tried and this works)